app/assets/javascripts/modules/nested-facets.js

```diff
@@ -0,0 +1,94 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  'use strict'
+
+  function NestedFacets (module) {
+    this.module = module
+    this.options = this.instantiateOptions()
+  }
+
+  NestedFacets.prototype.init = function () {
+    if (this.module) {
```

Comment:
There shouldn't be a need to check for the existence of `this.module` - if the module has been initialised, something will have been passed to it.